cfme/tests/v2v/test_v2v_cancel_migrations.py

```diff
@@ -174,32 +174,27 @@ def _cleanup():
     migration_plan.wait_for_state("Started")
     request_details_list = migration_plan.get_plan_vm_list(wait_for_migration=False)
     vm_detail = request_details_list.read()[0]
+    time.sleep(360)  # grace time for starting disk migration
```

Comment:
Instead of this you can check if migration plan is in progress by doing 
migration_plan.wait_for_state("In_Progress") and then cancel